realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -255,9 +261,13 @@ private boolean containsDot(String s) {
      * field type
      * @throws java.lang.RuntimeException Any other error
      */
-    public RealmQuery<E> equalTo(String fieldName, long value) {
+    public RealmQuery<E> equalTo(String fieldName, Short value) {
```

Comment:
Are we sure we want this instead of people being forced to use `isNull` instead?